lib/code_fund/stats/user_impressions.ex

```diff
@@ -0,0 +1,103 @@
+defmodule CodeFund.Stats.UserImpressions do
+  use GenServer
+  use CodeFundWeb, :query
+  alias CodeFund.Schema.UserImpression
+
+  @refresh_interval :timer.minutes(60)
```

Comment:
I would maybe put this value in an env var so we can adjust if needed without a deployment and so we can adjust update intervals for test purposes.